src/pages/RequestCallPage.js

```diff
@@ -102,19 +99,23 @@ class RequestCallPage extends Component {
             return;
         }
 
-        const personalPolicy = _.find(this.props.policies, policy => policy && policy.type === CONST.POLICY.TYPE.PERSONAL);
-        if (!personalPolicy) {
+        // If there's a lastAccessedWorkspace in Onyx then use that as the policy for the call, otherwise use the personal policy
+        const policyForCall = this.props.lastAccessedWorkspace
+            ? this.props.lastAccessedWorkspace : _.find(this.props.policies, policy => policy && policy.type === CONST.POLICY.TYPE.PERSONAL);
+
+        if (!policyForCall) {
             Growl.error(this.props.translate('requestCallPage.growlMessageNoPersonalPolicy'), 3000);
```

Comment:
I read the copy for the `growlMessageNoPersonalPolicy`, and I'm wondering what a user should do in this instance. Should we suggest they go to old dot and make a personal policy?

Reply:
Yeah it is a bit of an odd scenario. That was something that was already there from before. I'm not sure how a user would even get into that scenario because it shouldn't be possible not to have a personal policy and there isn't a way to create one. It's just something every account has. 

It looks like @jasperhuangg added that [here](https://github.com/Expensify/App/pull/3773), though I'm not sure what the scenario would be where there is no personal policy. Perhaps we could update the error message to recommend that they reach out to Concierge to ask for assistance.

Reply:
+1 Feels like the user can do nothing in this case and our code is broken somehow. Maybe we should log an alert instead? Why is there no policy?

Separately, it seems totally impossible that they would have... no policies at all. So we should allow this request to be made regardless and then default to the personal policy in the back end if none was provided?

Reply:
Hm, so I looked into the web code that handles the creation of the job and it seems that everything should work fine even if we don't pass a policy ID. So I'm just going to allow the request to be made in this scenario.